specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/diskRestorePoint.json

```diff
@@ -340,7 +340,7 @@
           "description": "unique incarnation id of the source disk",
           "readOnly": true
         },
-        "encryption": {
```

Comment:
Was this an error in the swagger? Please review with Shrutika

Reply:
Reverted back after discussing with Shrutika